XRTK.SpatialPersistence/Packages/com.xrtk.spatial-persistence/Runtime/Interfaces/IMixedRealitySpatialPersistenceDataProvider.cs

```diff
@@ -40,7 +40,7 @@ public interface IMixedRealitySpatialPersistenceDataProvider : IMixedRealityData
         void TryCreateAnchor(Vector3 position, Quaternion rotation, DateTimeOffset timeToLive);
 
         /// <summary>
-        /// Create a cloud Anchor at a specified location
+        /// Create a cloud Anchor at a specified location async
```

Comment:
does adding async to the docs really make that big of a difference if the name of the method already has asycn in it?